converter.py

```diff
@@ -0,0 +1,75 @@
+import tensorflow as tf
```

Comment:
where does this script get installed? I think it should be added to the package.yaml/.cabal ?

Reply:
This should be installed using `pip3 install tensorflow`. It's a Python package, not a Haskell package. I'm not sure where we should add this, though?

Reply:
no, I meant, this file `converter.py`. where does stack/accelerate find it when the time comes? I didn't see it mentioned in the package.yaml so expect it isn't going to be copied to the right place when you `stack install`